telemetry/telemetry_record.py

```diff
@@ -73,15 +100,48 @@ def move_wpr_files(name_schema):
     """
     remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/',
             name_schema)
-    local_path = os.path.join(os.getcwd(), 'data/')
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source')
     p = Popen('mv {0} {1}'.format(remote_data_path, local_path), shell=True)
 
+def make_modified_json(num_urls):
+    """Creates *.json files to point to the right archive files
+
+    Located in ~/src/tools/perf/page_sets/data
+    :param num_urls: int for the number of target urls
+    TODO: add sha1?
+    """
+    json_path = '/home/jamshed/src/tools/perf/page_sets/data/'
+    json_template = ('{{'
+            '"description": "Describes the Web Page Replay archives for a user '
+            'story set. Dont edit by hand! Use record_wpr for updating.", '
+            '"archives": {{ "url{0}_page_set_000.wpr": ["url{0}"]}}}}\n')
+    for i in range(num_urls):
+        modified_name = '999999' + str(i)
+        file_name = 'url{0}_page_set.json'
+        with open('{0}{1}'.format(json_path,
+            file_name.format(modified_name)), 'wb') as f:
+            f.write(json_template.format(modified_name))
+
 def modify_wpr():
-    """Sets cacheable objects delay time to 0, creates  pc.wpr files"""
-    wpr_directory = os.path.join(os.getcwd(), 'data/')
+    """Sets cacheable objects delay time to 0, creates pc.wpr files
+
+    Note: *.pc.wpr files are converted to 999999 + str(url name), which lets
+    them be treated like regular urls. Just know that six 9's indicate a
+    modified wpr file
+    TODO: Fix the 999999 notation
```

Comment:
Again, I'd be happy to change this notation, I think the base64 may be the best option here. Here is a sample benchmark:

```
@benchmark.Enabled('android')                                                   
class PageCyclerUrl0(_PageCycler):                                              
    page_set = page_sets.url0PageSet 
```
Here is the page_set file for this: `url0.py` and the wpr file is: `url0_page_set_000_.wpr`.


I can't have a `.` in python classes (referring to putting .wpr in the class name?), and I referenced a url by its index (in target_urls), which is 0 here. As a hack, I put six 9's in front of the index to indicate a modified url, which makes it easy to pull out the real index and know that this page set has been modified. 

Do you think the best option here is to convert everything base64, use that as the class name? This would be a lot cleaner, but it would also make it human-unreadable to see which urls failed, etc, as there is no logical ordering.

It may be easier to discuss this in person? Let me know what you think.